SkillSwapHub/src/main/resources/static/js/logInUsers.js

```
@@ -17,30 +21,77 @@ async function submitLoginForm(event) {
-        console.log(error.message);
-        document.getElementById("message").innerHTML = `<p style="color: red;">Error: ${error.message}</p>`;
+        showMessage("Network error. Please check your connection.", "error");
     }
 }
 
+// Function to display validation error messages
+
+
+//Function to display general messages
+function showMessage(message, type) {
+    let toastContainer = document.getElementById("toast-container");
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'toastContainer' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)